dashboard/app/models/user.rb

```diff
@@ -816,6 +816,8 @@ def presence_of_email_or_hashed_email
   def email_and_hashed_email_must_be_unique
     # skip the db lookup if we are already invalid
     return if errors.present?
+    # allow duplicate accounts to be created for LMS users that are unlinked
+    return if authentication_options.length == 1 && authentication_options.first&.credential_type == AuthenticationOption::LTI_V1
```

Comment:
Is it possible for an account to having just an LTI auth option and then get another auth option (of any kind) which would suddenly forever invalidate the User?

Reply:
This would be happening today if it was the case (this PR relaxes the rule instead of making it stricter). If the user gets _another_ account linked it might not lead to the situation where the user is erroring (because this check only occurs if changing an email). However, I do think this is a case where we need to think about it further longer term and we'll need to discuss the account linking design further as a team.